hbp_validation_framework/__init__.py

```diff
@@ -180,6 +194,59 @@ def _hbp_auth(self, username, password):
         else:
             raise Exception("Something went wrong. Status code {} from NMPI, expected 302".format(rNMPI1.status_code))
 
+    def _translate_URL_to_UUID(self, path):
```

Comment:
This method should be in the `CollabDataStore` class.

Reply:
Will be done in https://github.com/apdavison/hbp-validation-client/issues/17